docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+    todoapp-dev:
+        build:
+            context: .
+            dockerfile: Dockerfile
+            target: development
+            
+        ports:
+            - "5100:5100"
+        
+        env_file:
+            - .env
+
+    todoapp-test-offline:
+        build:
+            context: .
+            dockerfile: Dockerfile
+            target: testing-offline
+        
+        env_file:
+            - .env.test
+        
+        volumes:
```

Comment:
We should also add an equivalent volume to `todoapp-dev` so that we can easily make development changes without stopping & restarting our dev container

Reply:
Added